test/test_compile_test_and_run.cpp

```diff
@@ -4,6 +4,8 @@
 //  Boost Software License, Version 1.0. (See accompanying file
 //  LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
+#define TEST_COMPLEX
+
 #include <boost/core/lightweight_test.hpp>
 #include <boost/cstdfloat.hpp>
```

Comment:
I have a hunch that cstdfloat.hpp is pulling in something that sets the policy defaults before instantiate.hpp gets included.

Reply:
Ok. We are moving fast in two threads, that is probably because I started two threads. So be it. 

We do, nonetheless, need to fix this properly. I've seen also your comments on style in the other thread. We can combine all these. But for some reason, I can't find the right syntax on the policies to make every compiler _happy_.